data/initiatives.yml

```diff
@@ -151,3 +145,10 @@ ios-needs:
   description: >-
     iOS builds & releases are more important than ever, and ensuring developers
     are unblocked and that releases can ship on time is very important.
+
+mozillavpn:
```

Comment:
Thanks! Can you run `generate.py` to rebuild the markdown?

Reply:
I had this ordered by roadmap priority. We may want to move this up, but let's not block on that.
Maybe we can resolve the initiative before that becomes an issue :)